torch/quantization/fx/lower_to_native_backend.py

```diff
@@ -0,0 +1,8 @@
+import torch
+
+def lower_to_native_backend(model: torch.fx.GraphModule):
+    """ Lower a quantized reference model (with reference quantized operator patterns)
+    to the native backend in PyTorch (fbgemm/qnnpack), both backends shares the same
+    operator signature so they can be lowered with the same function
+    """
+    pass
```

Comment:
Currently these functions are no-ops. So how do we go from reference to actual backend?

Reply:
sorry, the changes are lost somehow, I just updated the PR again